src/store.js

```diff
@@ -177,7 +199,7 @@ export const store = reactive({
 				console.error(err)
 			})
 	},
-	// @todo why does the following run through the store?
+	// @todo why does the following run through the store? -- because its impossible with props, and its vital information for the modal.

```

Comment:
Das niet hellemaal waar natuurlijk, op het moment dat je door de list heen gaat heb je al items. Anders had je de lsit niet kunnen printen.

Reply:
Dit gaat over de edit modal van de publication data properties, net zoals metadata properties.

Sinds naar mijn kennis een modal geen props accepteert zoals de detail page zou doen, moet ik de store gebruiken om de juiste informatie beschikbaar te maken naar de modal, net zoals je doet met `publicationItem`

En als de modal niet deze informatie krijgt, is het onmogelijk om te identificeren welke data veranderd moet worden in de publication data.

Misschien kan dit verbeterd worden in een toekomstige refactor.